crates/egui/src/data/input.rs

```diff
@@ -651,20 +651,48 @@ impl Modifiers {
         self.alt && self.ctrl && self.shift && self.command
     }
 
+    /// Is alt the only pressed button?
+    #[inline]
+    pub fn alt_only(&self) -> bool {
+        self.alt && !(self.ctrl || self.shift || self.command)
+    }
+
+    /// Is ctrl the only pressed button?
+    #[inline]
+    pub fn ctrl_only(&self) -> bool {
+        self.ctrl && !(self.alt || self.shift || self.command)
+    }
+
     /// Is shift the only pressed button?
     #[inline]
     pub fn shift_only(&self) -> bool {
-        self.shift && !(self.alt || self.command)
+        self.shift && !(self.alt || self.ctrl || self.command)
+    }
+
+    /// Is super the only pressed button?
+    #[inline]
+    pub fn super_only(&self) -> bool {
+        self.mac_cmd && !(self.alt || self.ctrl || self.shift)
+    }
+
+    /// true if only [`Self::mac_cmd`] is pressed on `MacOs`.
+    #[inline]
+    pub fn mac_cmd_only(&self) -> bool {
+        if !cfg!(target_os = "macos") {
+            return false;
+        }
```

Comment:
Can't use the compile-time `target_os` in egui, since that won't work on wasm32. That's why we have `ctx.os()`

Reply:
ctx.os() didn't seem to work here.
Let me check again.

Reply:
Maybe we don't need this check at all. Let's just define `mac_cmd` to mean `super`?

Reply:
Changed to

```rust
    pub fn mac_cmd_only(&self) -> bool {
        self.super_only()
    }
```